Client/src/Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -819,32 +817,33 @@ public ICollection<GetResultIdsResponse.Types.MapTaskResult> GetResultIds(IEnume
     {
       if (resultStatus.IdsError.Any() || resultStatus.IdsResultError.Any())
       {
-        var msg =
-          $"The missing result is in error or canceled. Please check log for more information on Armonik grid server list of taskIds in Error : [ {string.Join(", ", resultStatus.IdsResultError.Select(x => x.TaskId))}";
+        var ids = string.Join(", ",
+                              resultStatus.IdsResultError.Select(x => x.TaskId));
 
         if (resultStatus.IdsError.Any())
         {
           if (resultStatus.IdsResultError.Any())
           {
-            msg += ", ";
+            ids += ", ";
           }
 
-          msg += $"{string.Join(", ", resultStatus.IdsError)}";
+          ids += string.Join(", ",
+                             resultStatus.IdsError);
         }
 
-        msg += " ]\n";
-
         var taskIdInError = resultStatus.IdsError.Any()
-                              ? resultStatus.IdsError.First()
-                              : resultStatus.IdsResultError.First()
-                                            .TaskId;
-
-        msg += $"1st result id where the task which should create it is in error : {taskIdInError}";
-
-        Logger.LogError(msg);
-
-        throw new ClientResultsException(msg,
-                                         resultStatus.IdsError.ToArray());
+                              ? resultStatus.IdsError[0]
+                              : resultStatus.IdsResultError[0].TaskId;
+
+        Logger.LogError("The missing result is in error or canceled. Please check log for more information on Armonik grid server list of taskIds in Error: [{taskList}]",
+                        ids);
+        Logger.LogError("1st result id where the task which should create it is in error : {taskIdInError}",
+                        taskIdInError);
```

Comment:
There two logs are redundant with the following error. There will have 3 logs in error with the same information

Reply:
The two log are not the same. I'll remerge them.